memory/src/main/scala/filodb.memory/data/ChunkMap.scala

```diff
@@ -130,9 +130,15 @@ object ChunkMap extends StrictLogging {
     if (numLocksReleased > 0) {
       logger.error(s"Number of locks was non-zero: $numLocksReleased. " +
         s"This is indicative of a possible lock acquisition/release bug.")
+      haltAndCatchFire
     }
     execPlanTracker.put(t, execPlan)
   }
+
+  def haltAndCatchFire(): Unit = {
+    logger.error(s"Shutting down process since it may be in an unstable/corrupt state.")
+    Runtime.getRuntime.halt(1)
```

Comment:
Can we use a unique exit code ?

Reply:
I suppose. What would it reveal that the log message didn't?

Reply:
Exit code is usually displayed in compute and  quicker to identify